### DISCUSSION TITLE
Auto-subscribe to discussions via upvote

### DISCUSSION THREAD

**Discussion by jsunsawyer** [community/community#4235] [Discussions]
I just upvoted a discussion with the expectation that I'd be subscribed to receive notifications, much like adding reactions to issues does.

It'd be nice if there was a notification setting in user preferences that changed this behavior.

The difference in behavior throws me off, as I'm not used to manually subscribing to notifications.

**Comment by markcellus** [comment]
> much like adding reactions to issues does

Can anyone confirm this? AFAICT I've never been auto-subscribed to a Github issue by just adding a reaction.

**Reply by jsunsawyer** [reply]
Oh gosh, just tested it myself and reactions don't seem to subscribe.
